src/tests/unit/common/store-tester.ts

```diff
@@ -74,15 +74,15 @@ export class StoreTester<TStoreData, TActions> {
         return actionsMock;
     }
 
-    private createActionMock() {
+    private createActionMock(): IMock<Action<{}>> {
         const actionMock = Mock.ofType(Action);
 
         actionMock.setup(a => a.addListener(It.is(param => param instanceof Function))).callback(listener => (this.listener = listener));
 
         return actionMock;
     }
 
-    private createChangeListener(store: IBaseStore<TStoreData>, times: Times) {
+    private createChangeListener(store: IBaseStore<TStoreData>, times: Times): IMock<(store, args) => void> {
```

Comment:
Just curious here: doesn't tslint gives a warning for the mocked function params: `store` and `args`? (I'm fine with this anyway)

Reply:
It doesn't and I am also curios as to why not 😃

Reply:
tslint is not warning because we have not enabled the `arrow-parameter` parameter on the `typedef` check. https://palantir.github.io/tslint/rules/typedef/

Reply:
Maybe we should enable this, but It will be nice to see what everyone's opinion on this is. Maybe plug this in the next Engineering meeting?

Reply:
Why don't we just add some argument types here in the spirit of the rules? Just my opinion.